lib/main.js

```diff
@@ -11,52 +11,14 @@ import NodePathTester from './node-path-tester';
 import JobManager from './job-manager';
 import * as helpers from './helpers';
 
-const IDLE_CALLBACKS = new Set();
-
-const makeIdleCallback = work => {
-  let callbackId;
-  const callback = () => {
-    IDLE_CALLBACKS.delete(callbackId);
-    work();
-  };
-  callbackId = window.requestIdleCallback(callback);
-  IDLE_CALLBACKS.add(callbackId);
-};
-
-const scheduleIdleTasks = () => {
-  const linterEslintInstallPeerPackages = () => {
-    require('atom-package-deps').install('linter-eslint-node');
-  };
-
-  if (!atom.inSpecMode()) {
-    makeIdleCallback(linterEslintInstallPeerPackages);
-  }
-};
-
-const SCOPES = atom.config.get('linter-eslint-node.scopes');
```

Comment:
I moved some stuff to the activate function since we could run into a race condition when the settings are not loaded before this file gets required but they are always loaded before the package is activated.